user_guide_src/add-edit-this-page

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env php
+<?php declare(strict_types=1);
+
+if ($argc === 1) {
+    echo 'Usage: ' . $argv[0] . ' <html-dir>' . PHP_EOL;
+    echo '  e.g: ' . $argv[0] . ' user_guide_src/build/html/' . PHP_EOL;
+
+    exit(1);
+}
+
+$dir = realpath($argv[1]);
+
+if ($dir === false) {
+    throw new RuntimeException('Cannot find directory: ' . $dir);
+}
+
+$iterator = new RecursiveIteratorIterator(
+    new RecursiveDirectoryIterator(
+        $dir,
+        FilesystemIterator::CURRENT_AS_FILEINFO |
+        FilesystemIterator::KEY_AS_PATHNAME |
+        FilesystemIterator::SKIP_DOTS
+    )
+);
+
+$files = new RegexIterator(
+    $iterator,
+    '/\A.+\.html\z/i',
+    RecursiveRegexIterator::MATCH
+);
+
+foreach ($files as $file_path => $file_info) {
```

Comment:
use camelCase for variables

Reply:
Why cs-fixer does not fix?

Reply:
I think rector would do this?

Reply:
I found https://github.com/rectorphp/rector/issues/3414

Reply:
Fixed.